src/App.tsx

```diff
@@ -50,15 +35,17 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
-                {!author && <p data-cy="NoSelectedUser">No user selected</p>}
+                {!selectedUser && (
+                  <p data-cy="NoSelectedUser">No user selected</p>
+                )}
 
-                {author && !loaded && <Loader />}
+                {selectedUser && loaded && <Loader />}
```

Comment:
The condition `selectedUser && loaded` for rendering the `Loader` component might be misleading. Typically, a loader is shown while data is loading, not after it has loaded. Consider revisiting this logic to ensure it reflects the intended behavior.